csm/cli/support_bundle/support_bundle.py

```diff
@@ -128,75 +97,84 @@ async def fetch_host_from_cluster():
             response_msg = {
                 "message": "No active nodes found. Cluster file may not be valid"}
             return Response(output=response_msg,
-                            rc=errors.CSM_ERR_INVALID_VALUE), None
+                            rc=CSM_ERR_INVALID_VALUE), None
         hostnames = []
         for each_node in active_nodes:
             hostnames.append(cluster_info.get(each_node, {}).get("hostname"))
         return hostnames, active_nodes
 
+    @staticmethod
+    def generate_bundle_id():
+        """Generate Unique Bundle ID."""
+        alphabet = string.ascii_lowercase + string.digits
+        return f"SB{''.join(random.choices(alphabet, k = 8))}"
```

Comment:
This is random. It may generate duplicate id, could be same as the last one. It would be better to append time string to the id?

Reply:
Actually we had tested this Algorithm while Developing Support Bundle the chances  for duplication is 1 in 10 lakhs. 
And as per earlier discussion time should not be a part for Bundle Id Hence never was added in this.

Reply:
Will there be a problem in generating support bundle if id gets generated same as previous one ?
If not, then we are fine else we should fix it even if chances are negligible.

Reply:
No There Won't Be any Problem. Except the Status Command will show the Data for Bundle Id if it is same.